packages/solid-query/src/__tests__/createQuery.test.tsx

```diff
@@ -5191,7 +5149,7 @@ describe('createQuery', () => {
           isFetching,
           isSuccess,
           isStale,
-        })
+        } as any)
```

Comment:
@lukesmurray @ardeora any idea why I had to make these changes here? What is the type `Optional` - I got errors that this is not found.

Also, the types we're trying to push into this Array here don't seem to be matching :/

Reply:
Hello! This is a bug that is caused by an issue upstream. I am writing a small PR to fix all of these any/Partial typings.

Reply:
thank you @ardeora

Reply:
No worries! #4308 aims to solve this and removes all the vague typings too